payroll/models/hr_payslip.py

```diff
@@ -523,29 +523,33 @@ def _get_salary_rules(self):
             sorted_rules |= rule_obj.browse(sorted_rule_ids)
         return sorted_rules
 
-    def _compute_payslip_line(self, rule, localdict, lines_dict):
+    def _compute_rule(self, rule, localdict, lines_dict):
```

Comment:
I think we should maintain `_compute_payslip_line` or if we change it, it should be different because `_compute_rule` is a method in salary rules. 

Do you find `_compute_payslip_line` not accurate?

Reply:
I agree. _compute_payslip_line is an accurate description of this method. Let's please keep it.